examples/zone-printer/README.md

```diff
@@ -0,0 +1,241 @@
+nodes in its Kubernetes cluster.
+
+The example also includes an Ingress manifest (`ingress/nginx.yaml`)
+that will be used by `kubemci`.
+
+# Tutorial
+
+## 0. Prerequisite
+
+### 0.1 GCP Project
+
+Ensure you that you have a GCP Project. If you don't have one already,
+you can create a project by following the instructions at
+[Creating and Managing Projects](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+
+After your project is created, run the following command in your
+terminal:
+
+```shell
+PROJECT=<your-project-name>
+```
+
+### 0.2 `gcloud` installed and initialized
+
+You can install `gcloud` by following the instructions at
+[Installing Cloud SDK](https://cloud.google.com/sdk/downloads) and
+initialize it by running the following commands:
+
+```shell
+gcloud init # Set $PROJECT as the default project
+gcloud auth login
+gcloud auth application-default login
+```
+
+### 0.3 kubectl
+
+You can install `kubectl` by running:
+
+```shell
+gcloud components install kubectl
+```
+
+### 0.4 Kubernetes clusters on GCP
+
+#### Existing clusters
+
+> **Note:**: If you don't already have Kubernetes cluster on GCP,
+skip to the next subsection on [creating new clusters](#creating-new-clusters).
+
+If you already have clusters with their credentials in your local
+kubeconfig (`$HOME/.kube/config` by default) file, you need to ensure
+that you either only have clusters you are going to use for this
+tutorial in your default kubeconfig file or create a new kubeconfig
+file that just contains credentials for those clusters. Assuming that
+the clusters you are going to use for this tutorial are called
+`us-east`, `eu-west` and `asia-east`.
+
+List the contexts available (note the names of the ones you wish to
+use with kubemci)
+
+```shell
+$ kubectl config get-contexts -o name
+us-east
+eu-west
+asia-east
+i-dont-care
+doesnt-exist
+```
+
+Now iterate over the contexts you identified, making it the context
+in-use and then extracting that context to its own file.
+
+```shell
+# First, do this for the us-east one.
+kubectl config use-context us-east
+kubectl config view  --minify --flatten > mciuseast
+
+# Second, do this for the eu-west one.
+kubectl config use-context eu-west
+kubectl config view --minify --flatten > mcieuwest
+
+# Finally, do this for the asia-east one.
+kubectl config use-context asia-east
+kubectl config view --minify --flatten > mciasiaeast
+```
+
+Combine (flatten) all of these extracted contexts to create a
+single kubeconfig file with just the contexts we care about
+
+```shell
+# Use KUBECONFIG to specify the context filenames we created in
+# the previous step
+KUBECONFIG=mciuseast:mcieuwest:mciasiaeast kubectl config view \
+  --flatten > mcikubeconfig
+```
+
+#### Creating new clusters
+
+You need at least two Kubernetes clusters in two different GCP zones to
+verify that `kubemci` works. Let's create three GKE clusters and get
+their credentials for the purposes of this tutorial:
```

Comment:
Should also note that they need at least v1.8.1

Reply:
Done.